lib/puppet/pops/validation.rb

```diff
@@ -145,14 +151,16 @@ def should_report? issue
     # @api private
     #
     def assert_issue issue
-      raise Puppet::DevError.new("Attempt to get validation severity for something that is not an Issue. (Got #{issue.class})") unless issue.is_a? Issues::Issue
+      unless issue.is_a? Issues::Issue
+        raise Puppet::DevError.new(_("Attempt to get validation severity for something that is not an Issue. (Got %{issue})") % { issue: issue.class })
+      end
     end
 
     # Checks if the given severity level is valid.
     # @api private
     #
     def assert_severity level
-      raise Puppet::DevError.new("Illegal severity level: #{option}") unless @@severity_hash[level]
+      raise Puppet::DevError.new(_("Illegal severity level: %{level}") % { level: level }) unless @@severity_hash[level]
```

Comment:
Note this option was changed to level as that is what we're checking.
P.S. No one calls this method

Reply:
Since it is not called, can we just remove it?

Reply:
Hah. Yeah, looks like the only place that was calling it was refactored to not call it any more in f61569a9c60162acc240105a529df1cf0cba344b. That sure does look like dead code.

Reply:
I didn't want to just remove it and break something. Also I wonder if we should add it somewhere, but we might find places where we're passing in unexpected values so I created another ticket for that. PUP-8304

Reply:
Alright I'm convinced I'll remove it.